packages/material-ui/src/IconButton/IconButton.js

```diff
@@ -103,7 +104,21 @@ IconButton.propTypes = {
   /**
    * The icon element.
    */
-  children: PropTypes.node,
+  children: chainPropTypes(PropTypes.node, props => {
+    React.Children.toArray(props.children).map(child => {
+      if (child.props.onClick) {
+        throw new Error(
```

Comment:
You need to return a single Error instance or nullish instead of throwing in propTypes.